### PR TITLE
net: refactoring

### PR DESCRIPTION
**Pull request by MaxVerevkin**
The `Net` struct has some fields that are marked as `Option` but they are always `Some`. Remove the option. Also print "N/A" when trying to get ssid or signal strength when wired connection is used instead of throwing an error.

TODO:
- [ ] Test on a machine with wired connection. (I don't have one)

### CODE REVIEW

**Review by ammgws — APPROVED** [review]

**Comment by ammgws** [comment]
> Maybe it's better to just print "N/A" if the connection is not wireless?

Sure, guess it's better than erroring the whole block/bar

File: src/blocks/net.rs [hunk 1]
```diff
@@ -888,10 +852,14 @@ impl Block for Net {
         let mut s_dn = self.shared_config.get_icon("net_down").unwrap_or_default();
         s_dn.push_str(&self.output_rx);
 
+        //let ssid = self.ssid.as_ref().unwrap_or(
```
Comment (ammgws) [inline src/blocks/net.rs:855]: Forgot to remove this?

Comment (MaxVerevkin) [reply src/blocks/net.rs]: Yeah :laughing:

**Comment by ammgws** [comment]
Works fine on my wired machine

**Comment by ammgws** [comment]
Thanks!

**Pull request merged**